index.js

```diff
@@ -120,7 +120,7 @@ var diff = function(input, output, path=[], options) {
 	if (typeof output === "undefined") {
 		var op;
 		if (json1) {
-			op = json1.removeOp(path, output);
+			op = json1.removeOp(path, input);
```

Comment:
Nice catch! I'd be curious to see if reversability holds across all the various ops generated.

I wonder if type.makeInvertible might come in handy for these - https://github.com/ottypes/json1/blob/ba9935f7c546c43f7609b780215e79fce9e01bd2/spec.md#inverting-operations